src/simulations/step.jl

```diff
@@ -32,22 +32,23 @@ dummystep(agent, model) = nothing
 until(ss, n::Int, model) = ss < n
 until(ss, n, model) = !n(model, ss)
 
-step!(model::ABM, agent_step!, n::Int=1) = step!(model, agent_step!, dummystep, n)
+step!(model::ABM, agent_step!, n::Int=1, agents_first::Bool=true) = step!(model, agent_step!, dummystep, n)
```

Comment:
I think this one, you need to pass `agents_first` through to the second function call yes?

Reply:
Yes, I missed that.